<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMap</class>
 <widget class="QMainWindow" name="TMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Карта</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IconsForMenu.qrc">
    <normaloff>:/1/Picture/Pic1.ico</normaloff>:/1/Picture/Pic1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Картографические проекции</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Визуализация</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>Астрономия</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_8"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="separator"/>
   <addaction name="action_5"/>
   <addaction name="action_4"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_9"/>
   <addaction name="action_10"/>
   <addaction name="separator"/>
   <addaction name="action_11"/>
   <addaction name="action_12"/>
  </widget>
  <widget class="QStatusBar" name="statusBarMain"/>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/WGS84.png</normaloff>:/1/Picture/WGS84.png</iconset>
   </property>
   <property name="text">
    <string>Цилиндрическая равнопромежуточная Плате-Карре</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/North.png</normaloff>:/1/Picture/North.png</iconset>
   </property>
   <property name="text">
    <string>Азимутальная ортографическая Северного Полюса</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/South.png</normaloff>:/1/Picture/South.png</iconset>
   </property>
   <property name="text">
    <string>Азимутальная ортографическая Южного Полюса</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/ShowAllMap.png</normaloff>:/1/Picture/ShowAllMap.png</iconset>
   </property>
   <property name="text">
    <string>Показать всю карту</string>
   </property>
  </action>
  <action name="action_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Arm.png</normaloff>:/1/Picture/Arm.png</iconset>
   </property>
   <property name="text">
    <string>Движение карты</string>
   </property>
  </action>
  <action name="action_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Ruler.png</normaloff>:/1/Picture/Ruler.png</iconset>
   </property>
   <property name="text">
    <string>Линейка</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/LayerCtrl.png</normaloff>:/1/Picture/LayerCtrl.png</iconset>
   </property>
   <property name="text">
    <string>Управление слоями</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Arrow.png</normaloff>:/1/Picture/Arrow.png</iconset>
   </property>
   <property name="text">
    <string>Информация по объектам на карте</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Mercator.png</normaloff>:/1/Picture/Mercator.png</iconset>
   </property>
   <property name="text">
    <string>Равноугольная цилиндрическая Меркатора</string>
   </property>
  </action>
  <action name="action_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Kavrayskiy.png</normaloff>:/1/Picture/Kavrayskiy.png</iconset>
   </property>
   <property name="text">
    <string>Псевдоцилиндрическая Каврайского</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Trass2Add.png</normaloff>:/1/Picture/Trass2Add.png</iconset>
   </property>
   <property name="text">
    <string>Показать</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="IconsForMenu.qrc">
     <normaloff>:/1/Picture/Trass2Hide.png</normaloff>:/1/Picture/Trass2Hide.png</iconset>
   </property>
   <property name="text">
    <string>Скрыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IconsForMenu.qrc"/>
 </resources>
 <connections/>
</ui>
